puppet-squid/src/main/java/com/iadams/sonarqube/puppet/api/PuppetGrammar.java

```diff
@@ -198,15 +193,7 @@ public static void grammar(LexerfulGrammarBuilder b) {
 
         b.rule(PARAMS).is(b.optional(
                 PARAM,
-                b.zeroOrMore(COMMA, PARAM))).skip();
```

Comment:
Ah OK. I completely missed de 'P' instead of the 'F' in front of 'ARROW"... I'm gonna fix it. Thanks.